docs/Guide/getting-started/creating-a-basic-app-command.mdx

```diff
@@ -54,15 +54,14 @@ export class PingCommand extends Command {
   }
 
   public override async chatInputRun(interaction: Command.ChatInputCommandInteraction) {
-    const msg = await interaction.reply({ content: `Ping?`, ephemeral: true, fetchReply: true });
+    await interaction.reply({ content: `Ping?`, flags: 'Ephemeral' });
+    const msg = await interaction.fetchReply();
```

Comment:
Can't we use `withResponse: true` instead? 👀

Reply:
Yes, that is also possible, but that returns a [InteractionCallbackResponse](https://discord.js.org/docs/packages/discord.js/14.19.1/InteractionCallbackResponse:Class) instead of a Message, which requires more code checks to satisfy TS.
So I checked the example Bots, and they seem to use this instead.

Reply:
I would prefer using withResponse. It really is not "more code" to have to do 1 object chain operation.

Reply:
Changed, tho I'm not sure if thats what you requested.

Reply:
As mentioned previously: new to TS and I just learned about the non-null assertion operator. Would it be bad practice to use it here?
```interactionCallbackResponse.resource!.message!```

Reply:
So I guess we can leave it as is, and merge it?